### PR TITLE
build: switch to npm

### PR DESCRIPTION
**Pull request by Wentao-Kuang**
#### Why
Moving to NPM to be more consistent across LINZ, most of LINZ uses npm and splitting the codebases between NPM and yarn is confusing and time consuming. Also the version of yarn we are using is deprecated and hasnt see significant improvements recently, where as npm has started getting a lot of new features.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,2432 @@
+{
+  "name": "basemaps-config",
+  "version": "1.0.1",
+  "lockfileVersion": 2,
```
Comment (blacha) [inline package-lock.json:4]: what version of npm/node are you using ? `v3` is the current lockFile version

Comment (Wentao-Kuang) [reply package-lock.json:4]: image
On v18.7

**Comment by blacha** [comment]
A description on why we are moving to NPM would be nice, how about?

### Why

Moving to NPM to be more consistent across LINZ, most of LINZ uses npm and splitting the codebases between NPM and yarn is confusing and time consuming. Also the version of yarn we are using is deprecated and hasnt see significant improvements recently, where as npm has started getting a lot of new features.

File: package-lock.json (hunk 1 above)
Comment (Wentao-Kuang) [reply package-lock.json]: Got my node updated and rerun the install. On v3 now.

**Review by blacha — APPROVED** [review]

**Review by blacha — APPROVED** [review]

**Pull request merged**